src/services/apiService.js

```diff
@@ -175,15 +176,72 @@ export async function GetSecretContent(Id, token) {
   return response.json();
 }
 
-/* Profile Page */
+/* User */
 
-export async function GetUserComments(username, commentsRepository) {
-  var query = `https://api.github.com/search/issues?q=is:issue+is:open+repo:${commentsRepository}+commenter:${username}`;
-  const response = await fetch(query, {
+export async function GetUser(userId, token) {
+  const response = await fetch(`${API_URL}/GetUserFunction?user_id=${userId}`, {
+    method: 'GET',
     headers: {
-      Authorization: `token ${GITHUB_API_PAT}`,
+      Authorization: `Bearer ${token}`,
     },
   });
+  return response.json();
+}
 
-  return await response.json();
+export async function ConnectUserCommentsAccount(data, token) {
+  const isEmpty = Object.values(data).some((x) => !x);
+  if (!data || isEmpty) {
+    return {
+      error: true,
+      message: 'Data is empty or in the wrong format',
+    };
+  }
+  await fetch(`${API_URL}/ConnectUserToCommentsFunction`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${token}`,
+    },
+    body: JSON.stringify(data),
+  });
+}
+
+export async function RemoveUserCommentsAccount(data, token) {
+  const isEmpty = Object.values(data).some((x) => !x);
+  if (!data || isEmpty) {
```

Comment:
put the isEmpty check in the if statement. I don't think it makes sense to use data in a statement prior to checking if it is defined to begin with